input/region_tail.xml.in

```diff
@@ -0,0 +1,7 @@
+
+      	</initialfacilitylist>
+      <config><NullInst/></config>
+    </institution>
+  </region>
+
```

Comment:
No problem. Since this file is xml, it's not really a pep8 compliance issue in this case. If it makes the final result more readable, feel free to keep it!